build/template-restore-build-MSIdentityWeb.yaml

```diff
@@ -17,6 +17,9 @@ steps:
     command: test
     projects: 'Microsoft.Identity.Web.sln'
     arguments: '--collect "Code coverage" --settings "build\CodeCoverage.runsettings" --configuration ${{ parameters.BuildConfiguration }} -p:RunCodeAnalysis=true -p:MsIdentityWebSemVer=${{ parameters.MsIdentityWebSemVer }} -p:SourceLinkCreate=true'
+  continueOnError: true
+
+- template: template-run-unit-tests.yaml
```

Comment:
I assume this is temporary ? "build restore template" is not for running tests.

Reply:
Yes. this was removed from here